chain/chain/src/validate.rs

```diff
@@ -222,13 +224,18 @@ fn validate_congestion_info(
             if header_congestion_info == &Some(CongestionInfo::default()) {
                 Ok(())
             } else {
-                Err(Error::InvalidCongestionInfo)
+                Err(Error::InvalidCongestionInfo(
+                    "Congestion Information invalid after protocol upgrade.".to_string(),
+                ))
```

Comment:
Similarly, would it make sense to include the header congestion info in the string?